Cargo.toml

```
@@ -17,48 +18,57 @@ members = ["tycho_simulation_py"]
 
 [dependencies]
 # Serialization/Deserialization
+chrono = { version = "0.4.26", features = ["serde"] }
+hex = "0.4.3"
 serde = { version = "1.0", features = ["rc"] }
 serde_json = "1.0.105"
-uuid = { version = "1.4.1", features = ["serde", "v4", "fast-rng", "macro-diagnostics"] }
-hex = "0.4.3"
-chrono = { version = "0.4.26", features = ["serde"] }
+uuid = { version = "1.4.1", features = [
+    "serde",
+    "v4",
+    "fast-rng",
+    "macro-diagnostics",
+] }
 
 # Error handling
 thiserror = "1"
 
 # Async & concurrency
-tokio = { version = "1.38.0", features = ["full"] }
 futures = "0.3.31"
+tokio = { version = "1.38.0", features = ["full"] }
 
 # Logging & Tracing
 tracing = "0.1.37"
 
 # Utility
-num-traits = "0.2.17"
 dotenv = "0.15.0"
-itertools = "0.10.5"
 enum_delegate = "0.2.0"
+itertools = "0.10.5"
+num-traits = "0.2.17"
 
 # Enum utilities
 strum = "0.25.0"
 strum_macros = "0.25.2"
 
 # Caching
-mini-moka = "0.10"
 lazy_static = "1.4.0"
+mini-moka = "0.10"
 
 # Tycho dependencies
-tycho-common = "0.83.3"
-tycho-client = "0.83.3"
+tycho-client = { git = "https://github.com/propeller-heads/tycho-indexer", version = "0.88.0" }
+tycho-common = { git = "https://github.com/propeller-heads/tycho-indexer",  version = "0.88.0" }
```

I think you can just do
```suggestion
tycho-common = "0.88.3"
tycho-client = "0.88.3"
```